sap/5.3/modules/ROOT/pages/sap-connector-reference.adoc

```diff
@@ -11,13 +11,25 @@ NOTE: Starting with version 5.0, SAP Connector also supports connecting to an on
 == Limitations
 
 * SNC is not supported for use in Mule apps that deploy SAP Connector in CloudHub and Anypoint Runtime Fabric.
-* Multiple applications with the same PROGRAM_ID, GATEWAY_SERVICE, and GATEWAY_HOST deployed on the same Mule runtime engine does not work because the JCoLibrary does not allow the creation of two servers connecting to instances that have the same PROGRAM_ID, GATEWAY_SERVICE, and GATEWAY_HOST. If you attempt to create two servers with the same PROGRAM_ID, GATEWAY_SERVICE, and GATEWAY_HOST, only one server is created. +
+* Multiple applications with the same PROGRAM_ID, GATEWAY_SERVICE, and GATEWAY_HOST deployed on the same Mule runtime engine does not work because the JCoLibrary does not allow the creation of two servers connecting to instances that have the same PROGRAM_ID, GATEWAY_SERVICE and GATEWAY_HOST. If you attempt to create two servers with the same PROGRAM_ID, GATEWAY_SERVICE, and GATEWAY_HOST, only one server is created. +
 Workarounds:
 . Use two different Mule runtime engines.
 . Use the same PROGRAM_ID but with two different GATEWAY_HOSTs.
+. Starting with SAP Connector 5.3.0 this feature is supported and the user can add multiple flows with multiple sources using the same PROGRAM_ID, GATEWAY_SERVICE and GATEWAY_HOST.
 
 * SAP Connector does not support Bitronix Transaction Manager. 
 
+* The reconnection strategy on SAP Listeners(Function Listener, Document Listener) relies exclusively on the JCO library which implies that the MULE mechanism for reconnections has no impact.
+* The JCO Server that runs  under the hood, in case of a connection issue it will manage the reconnection in the following ways:
+
+ ** If a connection issue occurs, the JCo server retries immediately.
+
+ ** If the connection is not restored immediately, the JCo server waits for one second and retries.
+
+ ** If the connection is not restored after the second retry, the JCo server waits for two seconds and retries.
+
+ ** If the connection is still not restored, the JCo server retries at four seconds, and then again at eight seconds.
```

Comment:
after that there isn't retries? If no please explain it here too

Reply:
I have modified those "ways"  based on this information. https://wiki.scn.sap.com/wiki/display/ASJAVA/SAP+JCo+FAQs#:~:text=It%20tries%20to%20reconnect%20in,be%20set%20using%20the%20%2DDjco. Check Q9.